beta-prompts/beta_prompts/seaborn_analytics.py

```
@@ -0,0 +1,1413 @@
+            )
+
+            # Add statistics overlay
+            mean_val = category_data.mean()
+            std_val = category_data.std()
+            ax.axvline(mean_val, color=color, linestyle="--", linewidth=2, alpha=0.8)
+
+            # Add text annotation
+            ax.text(
+                mean_val,
+                ax.get_ylim()[1] * 0.9 - i * 0.15,
+                f"{category}\nμ={mean_val:.2f}, σ={std_val:.2f}",
```

Ruff (0.12.2)

String contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)